corehq/apps/app_manager/views.py

```diff
@@ -1265,6 +1268,8 @@ def form_designer(request, domain, app_id, module_id=None, form_id=None,
         vellum_plugins.append("commtrack")
     if toggles.VELLUM_SAVE_TO_CASE.enabled(domain):
         vellum_plugins.append("saveToCase")
+    if toggles.VELLUM_EXPERIMENTAL_UI.enabled(domain):
```

Comment:
Might be easier to have in vellum instead of here, and add databrowser to options.plugins if experimental_ui is enabled